src/uct/sm/mm/base/mm_iface.c

```diff
@@ -850,6 +850,28 @@ static UCS_CLASS_CLEANUP_FUNC(uct_mm_iface_t)
     ucs_arbiter_cleanup(&self->arbiter);
 }
 
+ucs_status_t
+uct_mm_iface_base_query_tl_devices(uct_md_h md,
```

Comment:
Why put this func specifically between `CLEANUP` and `DEFINE`?
Maybe move above, where the other helper funcs are?